modules/cas_cache/volume/vol_block_dev_bottom.c

```diff
@@ -112,27 +113,33 @@ static const char *__block_dev_get_elevator_name(struct request_queue *q)
 }
 #endif
 
+*/
+
 /*
  *
  */
 const char *block_dev_get_elevator_name(struct request_queue *q)
 {
-	if (!q)
+	/*if (!q)
 		return NULL;
 
 	if (q->elevator == NULL)
 		return NULL;
 
 	return __block_dev_get_elevator_name(q);
+	*/
```

Comment:
Do we need it at all if it always returns NULL?

Reply:
Ah, yes. It is going to /dev/null, this one ;)